server/index.js

```diff
@@ -26,99 +27,101 @@ if (process.env.NODE_ENV === 'test') {
  * keys as environment variables, so that they can still be read by the
  * Node process on process.env
  */
-if (process.env.NODE_ENV !== 'production') require('../secrets')
+if (process.env.NODE_ENV !== 'production') require('../secrets');
 
 // passport registration
-passport.serializeUser((user, done) => done(null, user.id))
+passport.serializeUser((user, done) => done(null, user.id));
 
 passport.deserializeUser(async (id, done) => {
   try {
-    const user = await db.models.user.findByPk(id)
-    done(null, user)
+    const user = await db.models.user.findByPk(id);
+    done(null, user);
   } catch (err) {
-    done(err)
+    done(err);
   }
-})
+});
 
 const createApp = () => {
   // logging middleware
-  app.use(morgan('dev'))
+  app.use(morgan('dev'));
+  // helmet middleware
+  app.use(helmet());
```

Comment:
app.use(helmet()); added on line 48

Reply:
okay great, high in the middleware stack